src/cclib/parser/molproparser.py

```diff
@@ -154,28 +170,34 @@ def _parse_orbitals(self, inputfile, line):
         occnos = []
         line = next(inputfile)
 
-        # Besides a double blank line, stop when the next orbitals are encountered for unrestricted jobs
-        # or if there are stars on the line which always signifies the end of the block.
-        while line.strip() and (not "ORBITALS" in line) and (not set(line.strip()) == {'*'}):
+        # Besides a double blank line, stop when the next orbitals
+        # are encountered for unrestricted jobs
+        # or if there are stars on the line which always
+        # signifies the end of the block.
+        while (line.strip() and ("ORBITALS" not in line)
+               and (not set(line.strip()) == {'*'})):
 
-            # The function names are normally printed just once, but if symmetry is used then each irrep
+            # The function names are normally printed just once,
+            # but if symmetry is used then each irrep
             # has its own mocoeff block with a preceding list of names.
             is_aonames = line[:25].strip() == ""
             if is_aonames:
 
-                # We need to save this offset for parsing the coefficients later.
+                # We need to save this offset for parsing
+                # the coefficients later.
                 offset = len(aonames)
 
                 aonum = len(aonames)
                 while line.strip():
                     for s in line.split():
                         if s.isdigit():
                             atomno = int(s)
-                            atombasis[atomno-1].append(aonum)
+                            atombasis[atomno - 1].append(aonum)
                             aonum += 1
                         else:
                             functype = s
-                            element = self.table.element[self.atomnos[atomno-1]]
+                            element = self.table.element[
+                                self.atomnos[atomno - 1]]
```

Comment:
See later dangling delimiter comment; this was fine as it was.

Reply:
It then exceeds 80 characters